roles/openshift_provisioning_aws_vpc/meta/main.yml

```diff
@@ -0,0 +1,16 @@
+galaxy_info:
+  author: Jason DeTiberus
+  description: AWS VPC provisioning for OpenShift
+  company: Red Hat, Inc.
+  license: Apache License, Version 2.0
+  min_ansible_version: 2.2
+  platforms:
+  - name: Fedora
+    versions:
+    - 24
+    - 25
+  - name: EL
+    versions:
+    - 7
+  galaxy_tags: []
+dependencies: []
```

Comment:
I purposely left this out, since it goes against the Ops conventions, and opted to enforce the role dependencies at the openshift_provisioning_aws role.